packages/network-of-terms-catalog/catalog/queries/search/goudatijdmachine-straten.rq

```diff
@@ -7,26 +7,29 @@ PREFIX luc-index: <http://www.ontotext.com/connectors/lucene/instance#>
 PREFIX omeka: <http://omeka.org/s/vocabs/o#>
 PREFIX owl: <http://www.w3.org/2002/07/owl#>
 PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX vrank: <http://purl.org/voc/vrank#>
 
 CONSTRUCT {
     ?piduri a skos:Concept ;
         skos:prefLabel ?prefLabel ;
         skos:altLabel ?altLabel;
         skos:scopeNote ?scopeNote ;
         skos:related ?related_uri ;
-        skos:exactMatch ?exactMatch_uri .
+        skos:exactMatch ?exactMatch_uri ;
+        vrank:simpleRank ?score .
```

Comment:
@coret No `vrank:simpleRank` was being returned yet in the GTM query, so `luc:score` was ignored. I added `simpleRank` so results are actually returned in ranked order now.